Cargo.toml

```diff
@@ -62,7 +62,7 @@ members = [
 resolver = "2"
 
 [workspace.package]
-crane.name = "fedimint"
```

Comment:
This needs to be `crane.name`. The right fix in: https://github.com/fedimint/fedimint/pull/5659